tests/asm/Zvbb_smoke/Zvbb_smoke.S

```diff
@@ -0,0 +1,42 @@
+.global test
+test:
+    addi x1, x1, 1
+
+    li a0, 0x400                  
+    li a1, 0xc1                                           
+    vsetvl x0, a0, a1
+
+    lui x30, 1
+    auipc x31, 0
+
+    vadd.vi v0, v0, 10               
+    vadd.vi v2, v0, 11, v0.t
```

Comment:
non related.

Reply:
that's for ensuring the Zvbb operations have non-zero input data. Or should there be **vmv.v.x** instead?

Reply:
It makes sense to me.